src/environments/v1/README.md

```diff
@@ -0,0 +1,55 @@
+# v1 environment schema
+
+```yaml
+# locals allow you to re-use the same value in multiple
+# places in your schema file
+locals:
```

Comment:
Can these also be passed in when a command targeting the environment config is run? Or how might secrets for each component be modified per environment?

Reply:
There are examples of specifying secrets for each component down below. See `architect/app1` and `architect/app2`.